setup.py

```diff
@@ -45,8 +48,15 @@
 lib_dir = os.path.join(cspice_dir, 'lib')
 
 TEST_DEPENDENCIES = ['numpy>=1.8.0', 'pytest>=2.9.0', 'six>=1.9.0']
-DEPENDENCIES      = ['numpy>=1.8.0', 'six>=1.9.0']
-REQUIRES          = ['numpy', 'six']
+DEPENDENCIES = ['certifi>=2017.7.27.1', 'numpy>=1.8.0', 'six>=1.9.0']
```

Comment:
certifi should be moved to the if statement block below unless there is something I am missing

Reply:
Done